website/snapshots.yml

```diff
@@ -0,0 +1,31 @@
+- name: Home page dark theme
+  url: https://wix.github.io/Detox/
```

Comment:
I would expect that you use `http://localhost:3000/Detox` instead. Please think again for yourself – what we are testing: local changes or the production version? If we assume we **always** want to test a production version, then how is this script going to **prevent** you from entering visual regressions before it's too late (so that you'd need to revert). Well, that's why it should be a local version. 🤷‍♂️

Reply:
After more research into it and reading your notes, I agree with your point and made the suggested change. Testing the local version will allow the user to catch visual regressions before they are pushed to production and avoid any need to revert changes later on (which is the point of doing this regression testing). Thanks for the explanation here!